simulariumio/trajectory_converter.py

```diff
@@ -507,40 +139,35 @@ def to_JSON(self):
         Return the current simularium data in JSON format
 
         """
-        buffer_data = TrajectoryConverter._read_trajectory_data(self._data)
+        buffer_data = JsonWriter.format_trajectory_data(self._data)
```

Comment:
moved this functionality to JsonWriter, I think this function could still be convenient